src/rred_reports/redcap/interface.py

```diff
@@ -1,20 +1,63 @@
+from pathlib import Path
+
 import typer
+from loguru import logger
+
+from rred_reports.redcap.main import ExtractInput, RedcapReader
+from rred_reports.reports.interface import get_config
```

Comment:
Should we define `get_config` in `src/rred_reports/__init__.py` maybe?

Reply:
Yeah good call there, same with top_level_dir, I'll change that